app/main.py

```diff
@@ -1 +1,46 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills.extend(["JavaScript", "HTML", "CSS"])
+
+    def create_awesome_web_page(self) -> str:
+        print(f"{self.name} is creating a webpage...")
+        return "<h1>Hello world</h1>"
+
+
+class BackendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+            super().__init__(name)
+            self.skills.extend(["Python", "SQL", "Django"])
```

Comment:
Indentation issue: The lines inside the `__init__` method of the `BackendDeveloper` class are over-indented. They should be aligned with the method definition line to maintain consistency with the other classes.